k8sfv/calc_graph.go

```diff
@@ -85,7 +85,7 @@ func rotateLabels(clientset *kubernetes.Clientset, nsPrefix string) error {
 					panic(err)
 				}
 				ns_in.ObjectMeta.Labels["maturity"] = changeTo[ii]
-				ns_out, err := clientset.CoreV1().Namespaces().Update(ns_in)
+				ns_out, _ := clientset.CoreV1().Namespaces().Update(ns_in)
```

Comment:
Probably worth at least logging this error.  I don't know this code well enough to know if we should stop after an error

Reply:
Fixed